gpm_api/utils/geospatial.py

```diff
@@ -23,14 +28,114 @@
 # - rename file crop.py?
```

Comment:
Do you think it's worth renaming it crop.py? 
I would remove it ...

Reply:
Well there are also the `get_extent` methods which are exposed, so I would stick with `geospatial` :)